scalableminds/webknossos
# Allow users to see their time statistics

This PR enables the time statistics for non-admin users. If the user is not an admin, the view just displays the logged in user's time statistics.
Currently, the view can only be opened at `<base-url>/reports/timetracking` if you are not an admin.
There needs to be a link added to this view e.g. in the navbar.
### URL of deployed dev instance (used for testing):
- [https://allowuserstoseetimestats.webknossos.xyz](https://allowuserstoseetimestats.webknossos.xyz)

### Steps to test:
- Log in as a non-admin user.
- Open  `<base-url>/reports/timetracking`.
- The view should instantly display your statistics as the logged in user is chosen as a default user.

### Issues:
- fixes  #3905

------
